### DISCUSSION TITLE
飞天资源论坛 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#195] [模板讨论]
飞天资源论坛 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
2023-11-29 23:13:33
日志：Failed at 4/7 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Error while processing content unencoding: incorrect header check,
Request URL: https://feitianwu7.com/plugin.php?id=yinxingfei_zzza%3Ayinxingfei_zzza_post

==================
还是遇到老问题了。

**Reply by wjf0214** [reply]
复现不出来，看着提示是head的编码格式有问题？你手动看看你的`Accept-Encoding`是什么情况？
